contracts/WildcardsQV.sol

```diff
@@ -41,83 +61,113 @@ contract WildcardsQV is Initializable {
     ////////////////////////////////////
     //////// SETUP CONTRACT////////////
     //// NOTE: Upgradable at the moment
-    function initialize(uint256 _votingInterval) public initializer {
+    function initialize(uint256 _votingInterval, ILoyaltyToken _addressOfLoyalyTokenContract, IWildCardToken _addressOfWildCardTokenContract, IWildCardSteward _addressOfWildCardStewardContract, address _addressOfDragonCard) public initializer {
         admin = msg.sender;
         votingInterval = _votingInterval;
 
         proposalDeadline = now.add(_votingInterval);
+
+        loyaltyToken = _addressOfLoyalyTokenContract;
+        wildCardToken = _addressOfWildCardTokenContract;
+        wildCardSteward = _addressOfWildCardStewardContract;
+        addressOfDragonCard = _addressOfDragonCard;
     }
 
     ///////////////////////////////////
     /////// Config functions //////////
     ///////////////////////////////////
     function changeVotingInterval(uint256 newInterval) external onlyAdmin {
         votingInterval = newInterval;
+        emit LogNewVotingInterval(newInterval);
+    }
+
+    function changeProposalState(uint256 _proposalId, bool newState) external onlyAdmin {
+        // state: true = Active, false = Withdrawn
+        if (newState) {
+            state[_proposalId] = ProposalState.Active;
+        }
+        else {
+            state[_proposalId] = ProposalState.Withdrawn;
+        }
+        emit LogProposalStateChange(_proposalId,newState);
     }
 
     ///////////////////////////////////
     /////// CreateProposal  ///////////
     ///////////////////////////////////
-    function createProposal(string calldata proposalHash)
+    function createProposal(address payable _addressOfCharity)
         external
         onlyAdmin
         returns (uint256 newProposalId)
     {
         // So the first proposal will have an ID of 1
         proposalId = proposalId.add(1);
```

Comment:
I guess we should change this so the first proposal has an ID of 0, in this way it will match the orgs id's from 0 to 3 in the wildcards contracts?

Reply:
done :)